test/Yandex/Allure/Adapter/Support/StepSupportTest.php

```diff
@@ -48,7 +50,7 @@ public function testExecuteStepCorrectly()
     /**
      * @expectedException Exception
      */
-    public function testExecuteFailingStep()
+    public function te1stExecuteFailingStep()
```

Comment:
Change this back to testExecuteFailingStep

Reply:
fixed